src/main/scala/io/digitallibrary/bookapi/service/ContentConverter.scala

```diff
@@ -45,6 +48,7 @@ trait ContentConverter {
 
             val forSmallDevice = image.appendElement("img")
             forSmallDevice.attr("src", fullSizeUrl)
+            forSmallDevice.attr("crossorigin", "anonymous")
```

Comment:
For cloudinary images the bytes transferred will probably be so low that we should drop the the different sizes based on the viewport. This makes things easer for us down the line when we add offline support, as the same image is rendered on all viewport sizes